java/Anton-Kozyk/src/com/zoolatech/lecture4/tasks/_2/BinaryTree.java

```diff
@@ -0,0 +1,59 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class BinaryTree {
+    private Node root;
+
+    static class Node {
```

Comment:
class Node can be private - we don't want smb else from outside of the tree to access it